app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java

```diff
@@ -70,6 +70,15 @@ public String saveCurrentUsername(String username) {
         return null;
     }
 
+    public String saveCurrentSchoolId(String schoolId) {
+        SharedPreferences.Editor editor = mPref.edit();
+        editor.putString("currentSchool", schoolId);
+
+        if (editor.commit())
+            return schoolId;
+        return null;
```

Comment:
I think you're right, we should refactor some things in general. But for @JonasWanke it fits for now ;)